### ISSUE TITLE
dokanfuse.rc: digit exceeds base

### ISSUE THREAD

**Issue by taruti** [dokan-dev/dokany#383]
Noticed on appveyor build output:

```
C:/projects/dokany/dokan_fuse/src/dokanfuse.rc:19: digit exceeds base
C:/projects/dokany/dokan_fuse/src/dokanfuse.rc:20: digit exceeds base
```

**Comment by Rondom** [comment]
Yep, the last part of the version number, gets interpreted as an octal number, because it has a leading zero since the version was switched from 1.0.0.5bbb to 1.0.1.0bbb (with leading zeroes). Will be "fixed" temporarily when we has the first 1.0.1 RC, because the number ist 1.0.1.1bbb :-D

**Comment by Liryna** [comment]
Warning fixed with the 1.0.1 release like @Rondom said 👍 https://ci.appveyor.com/project/Maxhy/dokany/build/1.0.1-2/job/39ahfpjvn5bs8ol5
I have also reset the build number in appveyor
